src/sentry/integrations/source_code_management/commit_context.py

```diff
@@ -67,6 +79,67 @@ def get_commit_context_all_frames(
         """
         return self.get_blame_for_files(files, extra)
 
+    def create_or_update_comment(
+        self,
+        repo: Repository,
+        pr_key: str,
+        comment_body: str,
+        pullrequest_id: int,
+        issue_list: list[int],
+        metrics_base: str,
+        comment_type: int = CommentType.MERGED_PR,
+        language: str | None = "not found",
```

Comment:
👍 to this, and also:

What meaning does a `None` value have? Is it distinct from "not found"? If so, maybe we can clarify that. Otherwise, I'd suggest making it non-nullable. Alternatively, we could make `None` the default and then do `(language or "not found")` at the appropriate place below.

Reply:
i like `(language or "not found")`, added that in